index.html

```diff
@@ -97,5 +97,212 @@ <h2 class="more_det">LET’S CONNECT</h2>
         </li>
       </ul>
     </section>
+    <section class="work-card">
```

Comment:
- [ ] As the previous reviewer pointed out, the requirements explictly tell you that there should be a container for the works section with a property of `display: grid`, and each project card should have its elements positioned using Flexbox.
You should end up with something like 
```
< section class="display_grid_class">
    < div class="work_card_flex_class"> Word card 1 </div>
    < div class="work_card_flex_class"> Word card 2 </div>
    ...
< /section>
```
Please make sure to address this last issue and your project will be ready for approval! 🎉